src/list.js

```diff
@@ -0,0 +1,137 @@
+/* eslint-disable */
+import 'lodash';
+import dots from '../assets/images/dots.png';
+import removeIcon from '../assets/images/delete.png';
+import { MakeComplete } from '../modules/makeComplete.js';
+
+class DoList {
+  constructor(description, completed = false, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+// Remove Function
+function removeTask(e) {
+  const itemClass = e.target.className;
+  const itemId = itemClass.split('-');
+  const id = parseInt(itemId[1], 10);
+  let getData = JSON.parse(localStorage.getItem('dolist'));
+  getData = getData.filter((task, index) => index !== id);
+  e.target.parentNode.remove();
+  getData.forEach((task, i) => task.index = i + 1);
+  localStorage.setItem('dolist', JSON.stringify(getData));
+  displayTask();
+};
+// Update Function
+const UpdateTask = (e) => {
+  const taskid = e.target.id;
+  const itemId = taskid.split('-');
+  const id = parseInt(itemId[1], 10);
+  const getData = JSON.parse(localStorage.getItem('dolist'));
+  const input = document.querySelector(`#activity-${id}`);
+  getData[id].description = input.value;
+  localStorage.setItem('dolist', JSON.stringify(getData));
+  displayTask();
+};
```

Comment:
- [ ] Writing the function in a way that each function performs one task and only one task alone makes your code much more manageable and dependency-free. I suggest you make the functions exclusive to their tasks e.g. if you write the `UpdateTask` function it should only include code relevant to updating the item. Displaying the items should be done in a separate function.